tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -242,7 +242,8 @@ public void testUnixSocket() throws Exception
         };
         try (DistributionTester.Run run1 = distribution.start(args1))
         {
-            assertTrue(run1.awaitFor(5, TimeUnit.SECONDS));
+            // Give it time to download the dependencies
```

Comment:
well we should avoid download it as it's already here locally. I will make a separate pr to fix that